### PR TITLE
added typpx.1.4.2

### PR DESCRIPTION
**Pull request by camlspotter**

### CODE REVIEW

File: packages/typpx/typpx.1.4.2/opam [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+  [ "ocaml" "setup.ml" "-configure" "--prefix" prefix ]
+  [ "ocaml" "setup.ml" "-uninstall" ]
+]
+depends: [
+  "ocamlfind" { build }
+  "omake" { build & < "0.10" }
+  "ppxx" { >= "2.3.0" & < "2.4.0" }
+]
+available: [
+  ocaml-version >= "4.06.0" & ocaml-version < "4.07.0"
```
Comment (jpdeplaix) [inline packages/typpx/typpx.1.4.2/opam]: Same thing as your other packages. Why this constraint ? It seem to compile fine at least on ```4.05.0```.

Comment (camlspotter) [reply packages/typpx/typpx.1.4.2/opam]: Wrong tagging.  typpx.1.4.2 should be compilable only with ocaml.4.06.0.

**Comment by avsm** [comment]
The logs are full of warnings about missing `META` files

```
- ocamlfind: [WARNING] No such file: /home/opam/.opam/4.06.0/lib/ppx_curried_constr/META
```

Are you sure they are installed correctly such that the resulting library is usable by other ocamlfind packages?

**Comment by camlspotter** [comment]
I cannot access the logs right now https://ci.ocamllabs.io : connection is refused...  I guess the warnings for the META files are produced when the install script tries to execute `ocamlfind remove <package>` for non existing ones.

**Pull request merged**